src/bloqade/task/batch.py

```diff
@@ -300,42 +353,18 @@ def get_errors(self, task_indices: Union[List[int], int]):
             )
         )
 
-    def __str__(self) -> str:
-        output = ""
-        for task_index, task_error in self.task_errors.items():
-            output += (
-                f"Task {task_index} failed to submit with error: "
-                f"{task_error.exception_type}\n"
-                f"{task_error.stack_trace}"
-            )
-
-        return output
-
-
-@BatchErrors.set_serializer
-def _serialize(self: BatchErrors) -> Dict[str, List]:
-    return {
-        "task_errors": [
-            (task_number, task_error)
-            for task_number, task_error in self.task_errors.items()
-        ]
-    }
-
-
-@BatchErrors.set_deserializer
-def _deserialize(obj: dict) -> BatchErrors:
-    return BatchErrors(task_errors=OrderedDict(obj["task_errors"]))
```

Comment:
why did you delete this code? this is making the CI fail. Please add this code back.

Reply:
I realize I made a mistake while merging conflicts. I was updating the docstrings as part of the unitary hack issue. I will resolve it. Thanks for correcting me.

Reply:
No problem, if you need help setting up your development environment take a look at our docs page on that: https://bloqade.quera.com/latest/contributing/developing-bloqade/

Reply:
@shubhusion can I close this PR? I haven't seen any updates on it